web/template/watch.gohtml

```diff
@@ -28,14 +28,14 @@
                 To view this video please enable JavaScript.
             </p></noscript>
         <div id="watchContent"
-             class="transition-all motion-reduce:transition-none duration-300 ease w-full pt-3 px-2 md:px-8 {{if $course.ChatEnabled}}md:w-4/6{{end}} lg:w-8/12 2xl:max-w-screen-xl mx-auto mx:h-full">
+             class="transition-all motion-reduce:transition-none duration-300 ease w-full pt-3 px-2 md:px-8 mx-auto mx:h-full">
```

Comment:
Since we increase player size we could use `p-2` instead of `pt-3 px-2 md:px-8`.

Reply:
Yep, that looks more uniform. :+1: I think this is fine for now, but in the future we could make the alignment really seamless like it is on Twitch e.g.

Reply:
What do you think about `p-4`? Then it looks a bit better when using the smaller player, otherwise the gap to the navbar seems to narrow.

Reply:
Maybe you can do a follow-up after we merged https://github.com/joschahenningsen/TUM-Live/pull/320, so it looks like the player snaps to all edges when using the theater mode? I am not that good with design. This would also need some changes to the alignment of the description and icons.

Reply:
A follow-up sounds like a good plan. Will do.